test/e2e/vsphere_workload_api_flux_test.go

```diff
@@ -0,0 +1,220 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func runSimpleMulticlusterFlow(test *framework.MulticlusterE2ETest) {
+	test.CreateManagementCluster()
+	test.RunInWorkloadClusters(func(wc *framework.WorkloadCluster) {
+		wc.CreateClusterWithAPIGitOps()
+		wc.DeleteClusterWithAPIGitOps()
+	})
+	test.ManagementCluster.StopIfFailed()
+	test.DeleteManagementCluster()
+}
+
+func TestVSphereMulticlusterWorkloadClusterAPIGitHubFlux(t *testing.T) {
+	vsphere := framework.NewVSphere(t)
+	managementCluster := framework.NewClusterE2ETest(
+		t, vsphere,
+	)
+	managementCluster.WithClusterConfig(
+		api.ClusterToConfigFiller(
+			api.WithKubernetesVersion(v1alpha1.Kube123),
+			api.WithControlPlaneCount(1),
+			api.WithWorkerNodeCount(1),
+			api.WithStackedEtcdTopology(),
+		),
+		managementCluster.WithFluxGithub(),
+		vsphere.WithBottleRocket123(),
+	)
+	test := framework.NewMulticlusterE2ETest(t, managementCluster)
+	test.WithWorkloadClusters(
+		framework.NewClusterE2ETest(
```

Comment:
Does the Kubernetes version not need to be set here for the cluster config?

Reply:
That is now being done through `WithClusterConfig`, which generates the base cluster config and then updates it with some fillers